sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -79,11 +79,10 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
   ```
   "<your-service-principal-object-id>"
   ```
-- Use the returned credentials above to set **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenant) environment variables. The following example shows a way to do this in Powershell:
+- Use the returned credentials above to set **AZURE_CLIENT_ID** (appId) and **AZURE_TENANT_ID** (tenant) environment variables. The following example shows a way to do this in Powershell:
```

Comment:
Is this doc change sufficient in explaining to the user what do to without using a client_secret?

Reply:
should suffice , i removed trhe ENV form there